test/functional/legacy/fnamemodify_spec.lua

```diff
@@ -16,17 +14,27 @@ describe('filename modifiers', function()
 
     source([=[
       func Test_fnamemodify()
-        let tmpdir = resolve('/tmp')
+        if has('win32')
+          set shellslash
+          let tmpdir = expand('$LOCALAPPDATA/Temp')
+          call assert_equal(tmpdir, expand('$USERPROFILE/AppData/Local/Temp'))
+        else
+          let tmpdir = resolve('/tmp')
```

Comment:
$TMPDIR is passed to the tests, can it be used instead? It is local to the project directory. (`./Xtest-tmpdir`)

Reply:
Yes.